catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/LatestCommand.java

```diff
@@ -66,11 +66,11 @@ protected Object executeWithSubject() throws Exception {
     console.printf(formatString, "", "", "", "");
     printHeaderMessage(String.format(formatString, NUMBER, ID, DATE, TITLE));
 
-    Filter filter = filterBuilder.attribute(Metacard.MODIFIED).before().date(new Date());
+    Filter filter = filterBuilder.attribute(Core.METACARD_MODIFIED).before().date(new Date());
```

Comment:
✏️ I know it's not part of your changes, but this command could be updated to use [`ShellTable`](https://github.com/apache/karaf/blob/karaf-4.2.1/shell/core/src/main/java/org/apache/karaf/shell/support/table/ShellTable.java):
```java
final ShellTable shellTable = new ShellTable();
shellTable.column(NUMBER);
shellTable.column(ID);
shellTable.column(DATE);
shellTable.column(TITLE).maxSize(MAX_LENGTH);
...
shellTable.addRow().addContent(i, metacard.getId(), postedDate, title);
```